auth/auth0/README.md

```diff
@@ -0,0 +1,82 @@
+# auth0-authentication
+
+Add Auth0 Authentication to your Graphcool project 🎁
+
+
+## Getting Started
+
+### 1. Add the template to your Graphcool service
+
+```sh
+graphcool add-template auth/auth0
```

Comment:
The correct command is `graphcool add-template graphcool/templates/auth/auth0` :wink: